cmd/create/cluster/cmd.go

```diff
@@ -1917,6 +1943,26 @@ func run(cmd *cobra.Command, _ []string) {
 		}
 	}
 
+	isSharedVPC := false
+	privateHostedZoneID := strings.Trim(args.privateHostedZoneID, " \t")
+	privateHostedZoneRoleARN := strings.Trim(args.privateHostedZoneRoleARN, " \t")
+	baseDomain := strings.Trim(args.baseDomain, " \t")
+	if privateHostedZoneID != "" ||
+		privateHostedZoneRoleARN != "" {
+		if privateHostedZoneID == "" ||
+			privateHostedZoneRoleARN == "" ||
+			baseDomain == "" {
+			r.Reporter.Errorf("To install a cluster into a shared VPC," +
```

Comment:
```suggestion
			r.Reporter.Errorf("To install a cluster into a shared VPC, " +
```

Reply:
I would suggest having the flags validation at the beginning of the command before the user goes through all the stages.

Reply:
make sense. And we can remove it directly once we have the interactive mode support later. Will make this change

Reply:
made the change